projects/demo/src/modules/components/accordion/accordion.component.ts

```diff
@@ -61,19 +64,23 @@ export class ExampleTuiAccordionComponent {
 
     size: TuiSizeS = this.sizeVariants[1];
 
+    constructor(
+        @Self() @Inject(TuiDestroyService) private readonly destroy$: Observable<void>,
```

Comment:
Please use `inject` everywhere

Reply:
Sure, next PR

because I want to make cherry picks in 3.x